src/root/providers/application.providers.ts

```diff
@@ -28,5 +31,8 @@ export const APPLICATION_PROVIDERS = [
   sessionValueProvider(cognitoTokenSession()),
   redirectRoutesValueProvider(redirectToRoutes),
   logoutFactoryProvider(cognitoLogoutAction, [SESSION_PERSISTENCE]),
-  refreshTokenFactoryProvider(cognitoRefreshTokenAction$, [HttpClient, COGNITO_PERSISTENCE, SESSION_PERSISTENCE])
+  refreshTokenFactoryProvider(cognitoRefreshTokenAction$, [HttpClient, COGNITO_PERSISTENCE, SESSION_PERSISTENCE]),
+  forwardBearerTokenFactoryProvider(
+    forwardBearerTokenInterceptorMaker(/\/api/, () => localStorage.getItem('aws.cognito.access-token'))
```

Comment:
Il y a deux choses que je pense qu'on peut améliorer ici :
- le fait de définir une fonction à la volée qui n'est pas très lisible
- le fait d'accéder au local storage avec une chaîne en dur dans un endroit non centralisé
```suggestion
    forwardBearerTokenInterceptorMaker(/\/api/, cognitoTokenSession())
```
`cognitoTokenSession()` te donne accès à un objet qui peut accéder à tous les éléments de la session stokés dans le local storage avec des fonction dédiées. `getAccess()` te permet de récupérer le "access token"

Reply:
Ah oui et on peut aussi mettre la regex `/\/api/` dans une constante `AUTHORIZED_ROUTES_PATTERN`

Reply:
👍